wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java

```diff
@@ -28,6 +28,8 @@
   public static final String JAVA_OUT_FLAG = "--java_out=";
   public static final String FILES_FLAG = "--files=";
   public static final String ROOTS_FLAG = "--roots=";
```

Comment:
I'll delete it completely, but not urgently 'cause now we have plugins that expect this to exist